grouper/models.py

```diff
@@ -652,7 +653,9 @@ class UserToken(Model):
     created_at = Column(DateTime, default=datetime.utcnow, nullable=False)
     disabled_at = Column(DateTime, default=None, nullable=True)
 
-    secret = Column(String(length=32), default=_make_secret, unique=True, nullable=False)
+    hashed_secret = Column('secret', String(length=64), unique=True, nullable=False)
```

Comment:
any reason not to rename the column to 'hashed_secret' as well? seems to make it self-documenting in the schema and more consistent with the code.